text/0014-network-time.md

```diff
@@ -338,9 +340,6 @@ To solve all the problems listed above a different way of handling `local time`
 
 How can the client send state so that it arrives in `server time` on the server? The client has to adjust `local time` so that it matches `server time` on arrival on the server. This is done by predicting the time at which the packet will arrive on the server. The client knows about the time when the last server packet arrived. To predict its `local time` it can add the `RTT` + a bit of buffering to that time and use that as the new `local time`. This concept of adjusting `local time` to the server is what we call `predicted time` from now on.
 
-
-![predicted time](0014-network-time/localTime.png)
-
```

Comment:
apparently, this image no longer exists in the branch — should we just get rid of this or are you planning to re-upload this image? @LukeStampfli